10up/distributor
# Pass arguments array to `apply_filters_deprecated()`.

### Description of the Change


As `apply_filters_deprecated()` includes parameters to throw a notice, the arguments for the filter are passed as an array of arguments.

Note: This may be clearer to review with [whitespace changes excluded](https://github.com/10up/distributor/pull/962/files?w=1).


Closes #960
Follow up to #961 

### How to test the Change


1. Install and activate Distributor
2. Install and activate Query Monitor
3. Create a new post.
4. Save

On Develop, Query Monitor will show a warning `in_array() expects parameter 2 to be array, string given`. 
On this branch the warning should no longer be thrown.

The warning comes from this line:

https://github.com/10up/distributor/blob/946db6c9433c1db04e6a89e6a78885de7fa5561c/includes/utils.php#L408

### Changelog Entry

> Fixed - N/A Please remove this entry, bug not released.


### Credits

Props @peterwilsoncc, @dkotter 


### Checklist:
